schutzbot/deploy.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -euxo pipefail
+
+# Get OS details.
+source /etc/os-release
+
+# Restart systemd to work around some Fedora issues in cloud images.
+sudo systemctl restart systemd-journald
+
+# Add osbuild team ssh keys.
+cat schutzbot/team_ssh_keys.txt | tee -a ~/.ssh/authorized_keys > /dev/null
+
+# Set up a dnf repository for the RPMs we built via mock.
+sudo cp osbuild-mock.repo /etc/yum.repos.d/osbuild-mock.repo
+sudo dnf repository-packages osbuild-mock list
+
+# Install the Image Builder packages.
+sudo dnf -y install composer-cli osbuild osbuild-ostree \
```

Comment:
Shouldn't it be enough to pull in `osbuild-composer-tests`? That can be fixed in a follow-up.

Reply:
I guess so, but I wanted to be explicit.

Reply:
Reason I suggest it is that it would help us catch missing `Requires` in our spec files.